tests/integration/server/location/location.service.test.ts

```diff
@@ -153,4 +160,53 @@ describe('Location Service', function () {
       });
     });
   });
+  describe('updateRoom', function () {
+    let result: RoomAdminResponse;
+    let response;
+    context('when trying to update a room that does not exist', function () {
+      beforeEach(async function () {
+        updateRoomRequest = {
+          id: '4193a3e5-5987-4083-97cf-a949c146260f',
+          name: '318a',
+          capacity: 35,
+        };
+        try {
+          result = await locationService
+            .updateRoom(updateRoomRequest.id, updateRoomRequest);
+        } catch (error) {
+          response = error;
+        }
+      });
+      it('should throw a Not Found exception', function () {
+        strictEqual(response.status, 404);
+      });
+    });
```

Comment:
Per note on the Service file, this should actually be looking for an `EntityNotFound` error, and the controller should be turning it into a `NotFoundException`.

(With that change this wouldn't be necessary, but there's also an `HttpStatus` enum in the `@nestjs/common` package that provides `HttpStatus.NOT_FOUND`, so you don't need to hard code `404`)